likeminds-chat-reactnative-ui/components/LMChatTextView/index.tsx

```diff
@@ -0,0 +1,34 @@
+import { StyleSheet, Text } from "react-native";
+import React from "react";
+import { LMChatTextViewProps } from "./types";
+
+export const LMChatTextView = ({
+  maxLines,
+  textStyle,
+  selectable,
+  onTextLayout,
+  children,
```

Comment:
consume other props as well for the text. Check Tagging TextInput for the reference